Description: Add the instruction to enable autojump to its manpage
    This modification is done in the source file manpage.md, then compiled with:
    pandoc -st man -o autojump.1 manpage.md
Author: Tanguy Ortolo <tanguy+debian@ortolo.eu>
Forwarded: not-needed
Last-Update: 2013-04-14
Index: autojump/docs/autojump.1
===================================================================
--- autojump.orig/docs/autojump.1
+++ autojump/docs/autojump.1
@@ -8,12 +8,26 @@ autojump is a faster way to navigate you
 It works by maintaining a database of the directories you use the most
 from the command line.
 .PP
+\f[I]autojump must be loaded before it can be used.\f[]
+.PP
 \f[I]Directories must be visited first before they can be jumped to.\f[]
 .SS USAGE
 .PP
 \f[C]j\f[] is a convenience wrapper function around \f[C]autojump\f[].
 Any option that can be used with \f[C]autojump\f[] can be used with
 \f[C]j\f[] and vice versa.
+.PP
+.fi
+.IP \[bu] 2
+Load autojump
+.RS 2
+.IP
+.nf
+\f[C]
+\&.\ /usr/share/autojump/autojump.sh
+\f[]
+.fi
+.RE
 .IP \[bu] 2
 Jump To A Directory That Contains \f[C]foo\f[]:
 .RS 2
Index: autojump/docs/header.md
===================================================================
--- autojump.orig/docs/header.md
+++ autojump/docs/header.md
@@ -9,6 +9,8 @@ DESCRIPTION
 autojump is a faster way to navigate your filesystem. It works by maintaining a
 database of the directories you use the most from the command line.
 
+*autojump must be loaded before it can be used.*
+
 *Directories must be visited first before they can be jumped to.*
 
 USAGE
@@ -17,6 +19,10 @@ USAGE
 `j` is a convenience wrapper function around `autojump`. Any option that can
 be used with `autojump` can be used with `j` and vice versa.
 
+- Load autojump:
+
+        . /usr/share/autojump/autojump.sh
+
 - Jump To A Directory That Contains `foo`:
 
         j foo
